tests/test_domain_cpp.py

```diff
@@ -996,6 +996,15 @@ def test_domain_cpp_ast_attributes():
     # position: parameters and qualifiers
     check('function', 'void f() [[attr1]] [[attr2]]', {1: 'f', 2: '1fv'})
 
+    # position: class
+    check('class', '{key}[[nodiscard]] Foo', {1: 'Foo', 2: '3Foo', 3: '3Foo', 4: '3Foo'},
+          key='class')
```

Comment:
```suggestion
    check('class', '{key}[[nodiscard]] Foo', {1: 'Foo', 2: '3Foo'}, key='class')
```

Reply:
The check will use the last ID defined, so no need to duplicate for newer versions.